internal/tools/go.mod

```diff
@@ -6,14 +6,40 @@ require (
 	github.com/client9/misspell v0.3.4
 	github.com/frapposelli/wwhrd v0.4.0
 	github.com/fzipp/gocyclo v0.3.1
+	github.com/go-critic/go-critic v0.5.6 // indirect
```

Comment:
There are a lot of indirect dependencies that we probably don't need to add. Can you review? If we need to add any, put them in a separate `require` block with a comment that explains the need

Reply:
Maybe it just needs a `go mod tidy`? I could be wrong but I also think different versions of golang add these differently too so it would be good to do that with the exact version we use now too (v1.15.11).

Reply:
> Maybe it just needs a go mod tidy?

We have a `go mod tidy` check on the CI that passed so I think that would not have solved it (IIRC things pinned with `// indirect` will always be kept by `go mod tidy`).

> PS: Top of the file here interestingly mentions 1.16 as the go version for this module file

This *should* not have any undesirable effects but I will check it and open a separate PR to address this if needed